features/include/pcl/features/impl/ppfrgb.hpp

```diff
@@ -125,7 +125,7 @@ pcl::PPFRGBRegionEstimation<PointInT, PointNT, PointOutT>::computeFeature (Point
   output.resize (indices_->size ());
   for (std::size_t index_i = 0; index_i < indices_->size (); ++index_i)
```

Comment:
Could be ranged based for loop?

Reply:
Sadly no (at least not trivially): `index_i` is used in line 171